### PR TITLE
some fixes

### PR DESCRIPTION
**Pull request by Ishan09811**

### CODE REVIEW

File: turnip_builder.sh [hunk 1]
```diff
@@ -148,12 +148,12 @@ port_lib_for_magisk(){
 	cat <<EOF >"meta.json"
 {
   "schemaVersion": 1,
-  "name": "Turnip - $date - $commit_short",
-  "description": "$mesa_version - $commit_short",
+  "name": "Mesa Turnip - $date - $commit_short",
```
Comment (Weab-chan) [inline turnip_builder.sh:151]: I think adding mesa here is not necessary... is there other turnip driver maker out there?

File: turnip_builder.sh [hunk 2]
```diff
@@ -148,12 +148,12 @@ port_lib_for_magisk(){
...
+  "description": "Vulkan $vulkan_version",
```
Comment (Weab-chan) [inline turnip_builder.sh:152]: I do agree the description could describe the driver a bit more

File: turnip_builder.sh [hunk 3]
```diff
@@ -148,12 +148,12 @@ port_lib_for_magisk(){
...
   "author": "mesa",
   "packageVersion": "1",
   "vendor": "Mesa",
-  "driverVersion": "Vulkan $vulkan_version",
+  "driverVersion": "$mesa_version",
```
Comment (Weab-chan) [inline turnip_builder.sh:156]: Following kimchi's lead, I think having the vulkan version here make sense..

**Review by Weab-chan — CHANGES_REQUESTED** [review]

**Comment by Weab-chan** [comment]
Would be great to provide a rationale for those change :)

**Comment by Ishan09811** [comment]
@Weab-chan 


no, kimchi is wrong because this place is for the mesa version not for the Vulkan version and you can see this when I use the driver version for the Vulkan version
Screenshot_20240303-130607

**Comment by Weab-chan** [comment]
> @Weab-chan
> 
> no, kimchi is wrong because this place is for the mesa version not for the Vulkan version and you can see this when I use the driver version for the Vulkan version Screenshot_20240303-130607

you do understand kimchi was in direct communication with the people contributing to turnip? I'm sure he was advised to do it this way too.. and your picture proves nothing

**Comment by Ishan09811** [comment]
@Weab-chan 
you didn't see the vVulkan 1.3.278 which is very weird?

**Comment by Weab-chan** [comment]
> @Weab-chan you didn't see the vVulkan 1.3.278 which is very weird?

Right that's a screenshot from skyline, you should add more context lol I'll change it to be just a number i guess, good catch!

**Comment by Ishan09811** [comment]
@Weab-chan 
you can use the driver version as mesa version and discription as the Vulkan version so it will be good

**Comment by Weab-chan** [comment]
Lets double check all emulators and make sure they look good

**Comment by Weab-chan** [comment]
Closed due to the latest changes

**Pull request closed without merging**